README.md

```diff
@@ -15,3 +15,8 @@ Issue content goes here
 
 /cc @twolfson
 ```
+
+## Attribution
+
+* Gear icon by [Jardson Almeida](https://thenounproject.com/search/?q=gear&i=102547)
```

Comment:
I could have sworn we needed to mention "from the Noun Project" in the attribution, is that no longer the case?